addons/calendar/data/mail_template_data.xml

```diff
@@ -367,86 +352,88 @@
-                                <t t-else=""> Join at</t><br/>
-                                <a t-att-href="object.videocall_location" target="_blank" t-out="object.videocall_location or ''">www.mycompany.com/calendar/join_videocall/xyz</a>
-                            </li>
-                        </t>
-                        <t t-if="object.location">
-                            <li>Location: <t t-out="object.location or ''">Bruxelles</t>
-                                <a target="_blank"
-                                    t-if="not is_online or is_online and object.location != object.appointment_type_id.location_id.name"
-                                    t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.location}}">(View Map)</a>
-                            </li>
-                        </t>
-                        <t t-if="recurrent">
-                            <li>When: <t t-out="object.recurrence_id.get_recurrence_name()">Every 1 Weeks, for 3 events</t></li>
-                        </t>
-                        <t t-if="not object.allday and object.duration">
-                            <li>Duration:
-                                <t t-out="('%dH%02d' % (object.duration,round(object.duration*60)%60))">0H30</t>
-                            </li>
-                        </t>
-                    </ul>
-                </td>
-            </tr>
-        </table>
-    </div>
-    <div class="user_input">
-        <hr/>
-        <p placeholder="Enter your message here"><br/></p>
-
+                    </t>
+                </span>
+            </div>
+            <div t-if="recurrent" style="margin-top: 5px;">
+                <span style="display: inline-block; width: 100px; opacity: 0.76;">When</span>
+                <t t-out="object.recurrence_id.get_recurrence_name() or ''">Every 1 Weeks, for 3 events</t>
+            </div>
+            <div t-if="not object.allday and object.duration" style="margin-top: 5px;">
+                <span style="display: inline-block; width: 100px; opacity: 0.76;">Duration</span>
+                <t t-out="('%dH%02d' % (object.duration,round(object.duration*60)%60))">0H30</t>
+            </div>
+            <div t-if="object.location" style="margin-top: 5px;">
+                <span style="display: inline-block; width: 100px; opacity: 0.76; vertical-align: top;">Location</span>
+                <div style="display: inline-block; max-width: 450px;">
+                    <t t-out="object.location or ''">Bruxelles</t>
+                    <a target="_blank"
+                        t-if="not is_online or is_online and object.location != object.appointment_type_id.location_id.name"
+                        t-attf-href="http://maps.google.com/maps?oi=map&amp;q={{object.location}}"
+                        style="margin-left: 12px; font-size: 12px; color: #008f8c; text-decoration: underline;">View Map</a>
```

Comment:
I don't know if this is for formatting reasons but it seems we pretty much never multiline in mail templates, so I would not here ^^